web/scripts/widgets.js

```diff
@@ -136,9 +136,11 @@ function addMultilineWidget(node, name, opts, app) {
 				left: `${t.a * margin + t.e}px`,
 				top: `${t.d * (y + widgetHeight - margin - 3) + t.f}px`,
 				width: `${(widgetWidth - margin * 2 - 3) * t.a}px`,
+				background: (!node.color || node.color==='#222')?'':node.color,
 				height: `${(this.parent.inputHeight - margin * 2 - 4) * t.d}px`,
 				position: "absolute",
-				zIndex: 1,
+				color: (!node.color || node.color==='#222')?'':'white',
```

Comment:
The styles can be edited by users so I don't think checking for a specific value like this is a good idea.

Reply:
This was trying to solve the issue that the black theme looks quite terrible with the colored boxes, I'll consider something else.